helm/chaos-mesh/templates/chaos-daemon-rbac.yaml

```diff
@@ -58,16 +58,45 @@ metadata:
     {{- include "chaos-mesh.labels" . | nindent 4 }}
     app.kubernetes.io/component: chaos-daemon
 spec:
-  privileged: false
-  allowPrivilegeEscalation: true
+  {{- if .Values.chaosDaemon.privileged }}
```

Comment:
I think we should keep this line same with L99:
```
{{ or (eq .Values.chaosDaemon.privileged true) (eq .Values.bpfki.create true) }}
```

We could create another variable like `chaos-daemon-need-privileged` to reuse that expression.

Reply:
Yep, good idea, done!